servers/audio/effects/eq.cpp

```diff
@@ -73,9 +74,9 @@ void EQ::recalculate_band_coefficients() {
 
 
 
-		double side_gain2=POW2(1.0/M_SQRT2);
-		double th=2.0*M_PI*frq/mix_rate;
-		double th_l=2.0*M_PI*frq_l/mix_rate;
+		double side_gain2=POW2(1.0/Math_SQRT2);
```

Comment:
Hmm... I guess I could do that. However I decided not to do so as there seems to be a set of non-standardized, but somehow common constants in C++ by now. So I thought I just added that to math_funcs.h. But if you force me to :v